src/main/java/net/tlabs/tablesaw/parquet/TablesawParquetWriter.java

```diff
@@ -36,54 +36,61 @@
 
 public class TablesawParquetWriter implements DataWriter<TablesawParquetWriteOptions> {
 
-  private static final Logger LOG = LoggerFactory.getLogger(TablesawParquetWriter.class);
+    private static final Logger LOG = LoggerFactory.getLogger(TablesawParquetWriter.class);
 
-  @Override
-  public void write(final Table table, final Destination dest) throws IOException {
-    throw new UnsupportedOperationException(
-        "The use of Destination is not supported, please use the write(Table, TablesawParquetWriteOptions) method");
-  }
-
-  @Override
-  public void write(final Table table, final TablesawParquetWriteOptions options)
-      throws IOException {
-    try (final ParquetWriter<Row> writer =
-        new Builder(new Path(options.outputFile), table)
-            .withCompressionCodec(CompressionCodecName.fromConf(options.compressionCodec.name()))
-            .withWriteMode(options.overwrite ? Mode.OVERWRITE : Mode.CREATE)
-            .build()) {
-      final long start = System.currentTimeMillis();
-      Row row = new Row(table);
-      while (row.hasNext()) {
-        row = row.next();
-        writer.write(row);
-      }
-      final long end = System.currentTimeMillis();
-      LOG.debug(
-          "Finished writing {} rows to {} in {} ms",
-          row.getRowNumber() + 1,
-          options.outputFile,
-          (end - start));
+    /**
+     * Register the TablesawParquetWriter in the default tablesaw registry.
+     * Writer is only associated with its options.
+     */
+    public static void register() {
```

Comment:
same registering story as for the reader

Reply:
Yes. Changed too